AmericanMedicalAssociation/ama-style-guide-2
# EWL-8823 series index page

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-8823: Series index page - Styleguide](https://issues.ama-assn.org/browse/EWL-8823)

## Description
Adds series index page to styleguide


## To Test
- cd insto styleguides folder (styleguides/ama-styleguide-2/styleguide) and run gulp serve to open pattern lab
- Under pages tab select "Series"
- Confirm styling matches the following Zeplins:
Desktop https://zpl.io/V4wkpp8
Tablet https://zpl.io/bW8XJej 
Mobile https://zpl.io/2p3Pe5y 

NOTE: If images not showing (namely new subscribe bell icon), exit gulp serve and re-run 'lando composer install', then repeat the steps above.

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
![Screen Shot 2021-05-12 at 12 20 06 PM](https://user-images.githubusercontent.com/67962801/118017608-70ed7b00-b31c-11eb-8a8f-16d185a63dc3.png)
![Screen Shot 2021-05-12 at 12 20 49 PM](https://user-images.githubusercontent.com/67962801/118017695-882c6880-b31c-11eb-81c2-344773413634.png)
![Screen Shot 2021-05-12 at 12 20 58 PM](https://user-images.githubusercontent.com/67962801/118017737-924e6700-b31c-11eb-94b9-65452942f374.png)

## Remaining Tasks
- N/A

## Additional Notes
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
